scripts/run-artifactory-container.sh

```diff
@@ -1,17 +1,25 @@
 #!/usr/bin/env bash
+set -x
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
-export ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION:-7.37.14}
-echo "ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION}"
+source "${SCRIPT_DIR}/get-access-key.sh"
+
+export ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION:-7.37.15}
+echo "ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION}" > /dev/stderr
 
 set -euf
 
 docker run -i -t -d --rm -v "${SCRIPT_DIR}/artifactory.lic:/artifactory_extra_conf/artifactory.lic:ro" \
-  -p8081:8081 -p8082:8082 -p8080:8080 releases-docker.jfrog.io/jfrog/artifactory-pro:${ARTIFACTORY_VERSION}
+  -p8081:8081 -p8082:8082 -p8080:8080 "releases-docker.jfrog.io/jfrog/artifactory-pro:${ARTIFACTORY_VERSION}"
+
+export ARTIFACTORY_URL=http://localhost:8081
 
 echo "Waiting for Artifactory to start"
-until curl -sf -u admin:password http://localhost:8081/artifactory/api/system/licenses/; do
-    printf '.'
+until curl -sf -u admin:password ${ARTIFACTORY_URL}/artifactory/api/system/licenses/; do
+    printf '.' > /dev/stderr
     sleep 4
 done
 echo ""
+
+# with this trick you can do $(./run-artifactory-container.sh) and it will directly be setup for you
+getAccessKey "${ARTIFACTORY_URL}" > /dev/null 2>&1
```

Comment:
??? You're throwing away the output? I would echo `export JFROG_ACCESS_TOKEN=` Then you can do as the trick above says.

Reply:
yeah, it worked because I was using `/dev/tty` in the function